code/datums/uplink/ammunition.dm

```diff
@@ -73,4 +73,76 @@
 	name = "K2557 Magazine"
 	item_cost = 2
 	path = /obj/item/ammo_magazine/super_heavy
-	desc = "A spare magazine, for the super heavy K2557 pistol."
\ No newline at end of file
+	desc = "A spare magazine, for the super heavy K2557 pistol."
+
+/datum/uplink_item/item/ammo/shotgun_slug
+	name = "Shotgun Slug"
+	item_cost = 2
+	path = /obj/item/ammo_casing/shotgun
+	desc = "A shotgun slug."
+
+/datum/uplink_item/item/ammo/shotgun_buckshot
+	name = "Buckshot Shell"
+	item_cost = 1
```

Comment:
ditto.

Reply:
Going to replace this with a buckshot ammo box later.

Reply:
Slug shot should be more expensive than buckshots, though. Keep slugs at 2 TC.